<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACMediaCycleOsgQt</class>
 <widget class="QMainWindow" name="ACMediaCycleOsgQt">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>829</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MediaCycleOsgQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="ACOsgCompositeViewQt" name="compositeOsgView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Media_Directory"/>
    <addaction name="actionLoad_Media_Files"/>
    <addaction name="actionLoad_ACL"/>
    <addaction name="actionLoad_XML"/>
    <addaction name="separator"/>
    <addaction name="actionSave_ACL"/>
    <addaction name="actionSave_XML"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionLoad_Config_File"/>
    <addaction name="actionSave_Config_File"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Config_File"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_Hide_Controls"/>
   <addaction name="actionLoad_Media_Files"/>
   <addaction name="actionLoad_Media_Directory"/>
   <addaction name="actionLoad_ACL"/>
   <addaction name="actionSave_ACL"/>
   <addaction name="actionClean"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Media_Directory">
   <property name="text">
    <string>Load Media Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionLoad_ACL">
   <property name="text">
    <string>Load Library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_ACL">
   <property name="text">
    <string>Save Library</string>
   </property>
   <property name="toolTip">
    <string>Save Library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_Media_Files">
   <property name="text">
    <string>Load Media Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionShow_Hide_Controls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Controls</string>
   </property>
   <property name="shortcut">
    <string>Alt+Tab</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionLoad_Config_File">
   <property name="text">
    <string>Load Config File</string>
   </property>
  </action>
  <action name="actionSave_Config_File">
   <property name="text">
    <string>Save Config File</string>
   </property>
  </action>
  <action name="actionEdit_Config_File">
   <property name="text">
    <string>Edit Config File</string>
   </property>
  </action>
  <action name="actionLoad_XML">
   <property name="text">
    <string>Load XML</string>
   </property>
  </action>
  <action name="actionSave_XML">
   <property name="text">
    <string>Save XML</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+@</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ACOsgCompositeViewQt</class>
   <extends>QWidget</extends>
   <header>ACOsgCompositeViewQt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
